### DISCUSSION TITLE
[Feature request] --silent option

### DISCUSSION THREAD

**Discussion by Folaht** [ankitcharolia/goenv#21] [Ideas]
I'm using goenv in a script and it's creating a lot of download output when I do an --install.

**Comment by ankitcharolia** [comment]
@Folaht  this is how it works. 

```
$ goenv --install 1.20.7
Installing Go version 1.20.7...
https://dl.google.com/go/go1.20.7.linux-amd64.tar.gz
95.58 MiB / 95.58 MiB [--------------------------------------------------------------------------------------------------------] 100.00% 11.75 MiB p/s 8.3s
Go version 1.20.7 is installed at /home/acharolia/.go/1.20.7.

```
```
$ goenv --use 1.20.7
Using Go version 1.20.7.
Please make sure to execute: source ~/.bashrc
```

```
$ source ~/.bashrc
```

```
$ go version
go version go1.20.7 linux/amd64
```
Could you please post that how you are downloading/installing ?
or maybe you have't downloaded the right `goenv` based on your CPU architecture (like arm, arm64, amd, amd64)
Kindly have a look here: https://github.com/ankitcharolia/goenv/releases/ 

Regards,

**Reply by Folaht** [reply]
I'm talking about having to do a 
`goenv --install 1.20.7 &>/dev/null` in a script to suppress the `95.58 MiB / 95.58 MiB [-...` output.
Many application have a `--silent` or `--quiet` option for this.  

`wget` and `curl` have it.

**Reply by ankitcharolia** [reply]
ohk. I have intentionally added progress bar in golang (using third party library) just to make it fancy :wink:

**Reply by Folaht** [reply]
It's really nice, except for in scripts, because that will produce the output of each step.
